### DISCUSSION TITLE
ItemizedLayer onItemSingleTapUp Bug

### DISCUSSION THREAD

**Discussion by agrossmann09** [mapsforge/vtm#1019] [Q&A] [answered]
I am currently following the example set in MarkerOverylayActivity and everything works well expect for the ItemGestureListener on the ItemizedLayer. I have tried multiple Bitmaps as well as several of the MarkerSymbol hotspot places but I am unable to click on an existing Marker.

**Comment by agrossmann09** [comment]
Update: I removed the gestureListener for on map clicks for plotting markers and now it works. I can't seem to get both a map even listener, in order to plot markers, and a listener for on marker events to work.

**Comment by devemux86** [comment, marked as answer]
Testing again the `MarkerOverlayActivity`, it seems to work properly.
- We can tap or long press on the markers
- And tap or long press on _empty map area_

<img src="https://user-images.githubusercontent.com/3484020/221114340-f4495f81-fac0-4db7-a64c-4b283d231e4b.jpg" width="50%"/>
